### ISSUE TITLE
Missing BatchNorm2D layer support

### ISSUE THREAD

**Issue by YukeWang96** [pytorch/opacus#319]
Hi, 

When I want to port the simple design to ImageNet Pytorch Training script, I encounter a problem that
```
 [NotImplementedError('grad sampler is not yet implemented for BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)'),
```
Is there any solution or workaround to this?

**Comment by karthikprasad** [comment]
Hello! BatchNorm2D is not compatible with DP and hence is unsupportable. You can check out https://opacus.ai/tutorials/guide_to_module_validator#Why-do-I-need-a-Module-Validator? for details.

A workaround is to use `GroupNorm`; Opacus provides an easy way to do this: 
```module = ModuleValidator.fix(module)```
See https://opacus.ai/tutorials/building_image_classifier#Model for a step-by-step tutorial.
